.golangci.yml

```diff
@@ -15,6 +15,7 @@ linters:
     # All these break for one reason or another
     - tagliatelle # too many JSON keys cannot be changed due to compat
     - nosnakecase # too many false positives due to the `unix` package
+    - dupword # too many false positives (e.g., in tests)
```

Comment:
we could exclude the test files for this linter

Reply:
Tests are just an example.  `dupword` seems to be checking any string and would also report things like `data := "0 0 0"` as a duplicate.  I don't think the linter is ready (for us).

Reply:
I don' think `dupword` is made for that use case.  It seems to only be looking for duplicates in strings and comments. "foo foo" for instance.

Reply:
oh you are right, I was thinking of `goconst`